include/alpaka/pltf/PltfHipRt.hpp

```diff
@@ -41,6 +42,12 @@ namespace alpaka
         };
     }
 
+    namespace queue
+    {
+        class QueueHipRtBlocking;
+        class QueueHipRtNonBlocking;
+    }
+
```

Comment:
Are those forward declarations necessary? You did not add them to the CPU and CUDA versions.

Reply:
no not needed. copy past issue